index.html

```diff
@@ -86,5 +86,197 @@ <h1>I'm Stephen Glad to see you</h1>
         </div>
       </section>
     </main>
+    <section class="project-section">
+      <div class="project-container flex column">
+        <div class="project-card flex column">
+          <img src="./images/Snapshoot Portfolio.svg" alt="project card image">
+          <div class="prject-card-info flex column">
+            <h2 class="project-title">Tonic</h2>
+            <div class="project-meta flex">
+              <p class="bold">Canopy</p>
+              <div class="round"></div>
+              <p>Backend</p>
+              <div class="round"></div>
+              <p>2015</p>
+            </div>
+            <div class="project-note">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="project-tag">
+              <ul class="project-skills flex">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JAVASCRIPT</li>
+              </ul>
+            </div>
+            <button>See Project</button>
+          </div>
+        </div>
+
+        <div class="project-card flex column">
+          <img src="./images/Snapshoot Portfolio (1).svg" alt="project card image">
+          <div class="prject-card-info flex column">
+            <h2 class="project-title">Tonic</h2>
+            <div class="project-meta flex">
+              <p class="bold">Canopy</p>
+              <div class="round"></div>
+              <p>Backend</p>
+              <div class="round"></div>
+              <p>2015</p>
+            </div>
+            <div class="project-note">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="project-tag">
+              <ul class="project-skills flex">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JAVASCRIPT</li>
+              </ul>
+            </div>
+            <button>See Project</button>
+          </div>
+         </div>
+
+          <div class="project-card flex">
```

Comment:
- [ ] Kindly increase the space between the cards to match the design 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/205633774-27d23d26-1591-4fdd-8818-8e2e589df8f3.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/205633823-ebc74472-776e-4662-b7c1-f858bdbc4fc9.png)